### ISSUE TITLE
How does it work?

### ISSUE THREAD

**Issue by gonutz** [itchyny/bed#26]
Please include a usage section in your readme, I like the idea of a console editor written in Go, I was recently looking for a good one and ended up writing my own.
The only problem is that I have no idea how to use your editor. Instead of the "why" section I would rather like to have a concise manual with all keyboard shortcuts and possible command line parameters explained.
Also I was trying to call bed with -h --help and /? which I think should all really print the same kind of help that should be in the readme.
Thanks very much for sharing this editor!

**Comment by itchyny** [comment]
This editor is influenced by Vim so most of the basic operation of Vim is supported, including modes. I don't list all the keyboard shortcuts in README.md but I'll add some notes for basic operations.

**Comment by gonutz** [comment]
Why don't you list all keyboard shortcuts in the readme? That is exactly where they belong, isn't it?
I am not at all familiar with vim as are most people on this planet. Why would you assume that someone who reads about this editor in the Go Newsletter knows anything about vim? Even the hardcore text editor people are split into emacs and vim camps! :-)
Please reconsider how essential it is to know how a program works. Do not alienate most of your potential userbase by not helping them out in the most basic way.

**Comment by itchyny** [comment]
Wow, linked from Go Newsletter? I noticed that now. How honored it is.
Note that README is not always the best place to list all the usage. README of both [Vim](https://github.com/vim/vim) and [Emacs](https://github.com/emacs-mirror/emacs) does not do it.
I'm thinking of creating embedded help which user can open with `:h` (which was listed in #1). But recently I don't have time for this project. Anyway, now that I wrote basic usages in README.md so closing.
